tests/manage/pv_services/test_rwo_pvc_fencing_unfencing.py

```diff
@@ -0,0 +1,632 @@
+import logging
+import pytest
+
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import ignore_leftovers, ManageTest, tier4a
+from ocs_ci.ocs import constants, node, ocp
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.ocs.resources import pod
+from ocs_ci.utility.retry import retry
+from ocs_ci.utility.utils import TimeoutSampler
+from subprocess import TimeoutExpired
+from tests import disruption_helpers, helpers
+
+logger = logging.getLogger(__name__)
+
+
+@ignore_leftovers
+class TestRwoPVCFencingUnfencing(ManageTest):
+
+    @pytest.fixture()
+    def setup(
+        self, request, scenario, num_of_nodes, num_of_fail_nodes,
+        disrupt_provisioner, project_factory, multi_pvc_factory, dc_pod_factory
+    ):
+        """
+        Create pvc and pods
+        """
+        num_of_app_pods_per_node = 2
+        worker_nodes_name, ocs_nodes_name, non_ocs_nodes_name = self.identify_nodes(
+            scenario, num_of_nodes
+        )
+
+        def finalizer():
+            if scenario == "colocated":
```

Comment:
please set the node list before calling helpers.remove_label_from_worker_node() so the function call will be written only once

Reply:
Done